### DISCUSSION TITLE
Error: Can't determine the local repo's default branch

### DISCUSSION THREAD

**Discussion by maryadejoh** [IBAHCM/RPiR#465] [Archive] [answered]
Hi @IBAHCM/rpir-instructors,
I am unable to connect to github from Rstudio when I run "usethis::use_github(organisation = "IBAHCM", private = TRUE)"
I get this error: Can't determine the local repo's default branch.

What could be the matter please?
Many thanks,#
Mary

**Comment by richardreeve** [comment, marked as answer]
@maryadejoh - you need to make an initial commit to the repository in the git pane in RStudio (just commit, don't push) before you run `use_github()`, otherwise it gets confused. Can you try that and see if it works?

**Reply by maryadejoh** [reply]
Thank you, Richard. This has been resolved.

**Comment by maryadejoh** [comment]
Thank you, Richard. I will try and see how it goes.

Regards,
Mary
